.github/workflows/ci.yml

```diff
@@ -21,4 +21,9 @@ jobs:
       run: cargo clippy --all-features -- -D warnings
     - run: cargo build --no-default-features --verbose
     - run: cargo test --features="master resolv resolv-sync sign tsig validate" --verbose
-
+    - if: matrix.rust == 'nightly'
+      run: |
+        cargo +nightly update -Z minimal-versions
+        cargo check --features="master resolv resolv-sync sign tsig validate" --verbose --all-targets
+        cargo test --features="master resolv resolv-sync sign tsig validate"
```

Comment:
Question related to these features: I think this is list is the same as `--all-features`, is there a reason to specify them separatedly?

Reply:
I think `interop` is missing. Which is important because that one wants NSD installed so it won’t work. If there’s a better way for selectively skipping tests than I’d be more than happy to change that.

Reply:
Aah ok, that explains it. I don't think there's a better way!